styled/resources.js

```diff
@@ -47,15 +47,23 @@ export const CategoryContainer = styled.View`
 `;
 
 export const CategoryIcon = styled.View`
-  flex: 1;
-  align-content: center;
+  display: flex;
   align-items: center;
   justify-content: center;
+  height: 40px;
+  width: 40px;
+  padding: 8px
```

Comment:
semicolon!

Reply:
(ESLint / prettier should catch this if you installed it on VS Code - @kennethlien I can walk you through it)